onnx/defs/tensor/defs.cc

```diff
@@ -3621,6 +3621,9 @@ ONNX_OPERATOR_SET_SCHEMA(
           std::vector<int64_t> axes;
           if (axes_attr) {
             axes = RetrieveValues<int64_t>(*axes_attr);
+            std::transform(axes.begin(), axes.end(), axes.begin(), [&](int64_t axis) -> int64_t {
```

Comment:
It would be useful to move this as a utility function into `onnx/defs/shape_inference.h`. Even though it is only 3 lines, it seems like something we would want to do in other similar situations. (Eg., I find the same in [Squeeze](https://github.com/onnx/onnx/blob/main/onnx/defs/tensor/defs.cc#:~:text=std%3A%3Atransform(axes,%7D)%3B)

Reply:
is there any validation of the range of the axis values?

Reply:
Done (and added some validation tools as well)